target/linux/ramips/dts/mt7620a_domywifi.dtsi

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		led-boot = &system_amber;
+		led-failsafe = &system_amber;
+		led-running = &system_green;
+		led-upgrade = &system_amber;
+		label-mac-device = &wmac;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio0 13 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		system_green: system_green {
```

Comment:
DT label should have led_ prefix, resulting in
`led_system_green: system_green {`
...

Reply:
fixed, I have added it.